HW_1/zadan_8.py

```diff
@@ -0,0 +1,8 @@
+rows = int(input('Введите количество рядов: '))
+
+for i in range(1, rows+1):
+    for j in range(1, (rows-i)+1):
+        print(' ', end='')
+    for k in range(1, (2*i)):
+        print('*', end='')
+    print()
```

Comment:
Не забывайте приводить к PEP8
```suggestion
rows = int(input('Введите количество рядов: '))

for i in range(1, rows + 1):
    for j in range(1, (rows - i) + 1):
        print(' ', end='')
    for k in range(1, (2 * i)):
        print('*', end='')
    print()

```

Reply:
Попробуйте обойтись 1 циклом